.sonar/report-task.txt

```diff
@@ -0,0 +1,5 @@
+projectKey=Profile_App
```

Comment:
.sonar/.sonar_lock
.sonar/report-task.txt

Above files should not be added to git